hypervisor/arch/x86/cpu.c

```diff
@@ -56,10 +56,10 @@ static bool init_percpu_lapic_id(void)
 	/* Save all lapic_id detected via parse_mdt in lapic_id_array */
 	pcpu_num = parse_madt(lapic_id_array);
 
-	if (pcpu_num != 0U) {
+	if (pcpu_num != 0U && pcpu_num <= CONFIG_MAX_PCPU_NUM) {
```

Comment:
In this API. we can directly use phys_cpu_num?

Reply:
No need to define pcpu_num.

Reply:
ok will use phys_cpu_num